pr_testing/test_multiple_prs.sh

```diff
@@ -786,6 +787,27 @@ if ! $CMSDIST_ONLY ; then # If a CMSSW specific PR was specified #
     exit 0
   fi
 
+  if [[ $PRODUCTION_RELEASE == "true" && ${PULL_REQUEST} == "*/cmssw#*" ]]; then
```

Comment:
@iarspider , looks like this check is still not working. Can you please test locally a simple script to test it before updating it here?